webportal/src/app/cluster-view/services/service-info.js

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation
+// All rights reserved.
+//
+// MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+// documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+// the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+// to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+// This function will call kubernetes restful api to get node - podlist - label info, to support service view monitor page.
+function getServiceView(kubernetesUrl) {
+  var Client = require('node-rest-client').Client;
+  var client = new Client();
+  var nodesInfo = []
+  var request = require('sync-request');
+  var resNodes = request('GET', kubernetesUrl + '/api/v1/nodes');
```

Comment:
why not use jquery’s .ajax() function, as yifan did in job-view.component.js?

Reply:
1. This is just a util function and example how to collect the restful data. In this function need make async to sync to construct the node-pods-labels collection, 
2. Who use this function could modify the function to their need and follow yifan's  job-view.component.js.

Reply:
I will refine this later.